tensorflow/python/data/experimental/kernel_tests/compression_ops_test.py

```diff
@@ -50,6 +60,22 @@ def _test_objects():
   ]
 
 
+def _test_eager_objects():
+  return [
```

Comment:
do these objects not work in graph mode?

Reply:
The tests raise a `RuntimeError: Attempting to capture an EagerTensor without building a function.` when I use 
```
@combinations.generate(
      combinations.times(test_base.graph_only_combinations(),
                         combinations.combine(element=_test_eager_only_objects())))
```
This is due to the `RaggedTensor` type.

Reply:
Got it, thanks!